### ISSUE TITLE
exporting - durationFormat and durationFields?

### ISSUE THREAD

**Issue by morrow95** [amcharts/amcharts5#139]
Came across another problem converting from version 4 to 5 and can't seem to figure it out.  In version 4 I was using this to make sure the 'Total' values were exported as duration and in 'h m s' format.  It doesn't look like version 5 supports these for exporting from what I see in the docs - or it's very possible I'm missing it.

For what it's worth this particular case is being used in a pie chart if that matters.

v4:
```
chartReg['chart'].exporting.durationFields.push("Total");
chartReg['chart'].exporting.durationFormat = "hh'h' mm'm' ss's'";
```

what I expected to work in v5:
```
exporting = am5plugins_exporting.Exporting.new(chartReg['chart'], {
	dataSource: response.data,
	filePrefix: selectTypeLabel + '-' + selectTypeText,
        durationFields: ["Total"],
        durationFormat: "hh'h' mm'm' ss's'"
});
```
doesn't seem to be the correct method.

Help or ideas?

**Comment by martynasma** [comment]
Thanks for the heads up. We'll add it in the next version.

We'll notify you here when that happens.

**Comment by martynasma** [comment]
Implemented in [5.0.16](https://github.com/amcharts/amcharts5/releases/tag/5.0.16).

## [5.0.16] - 2021-12-10

### Added
- `tooltipPositionX` and `tooltipPositionY` added to `XYSeries` (possible values: `"value"`, `"open"`, `"low"`, and `"high"`). Indicates which data item value tooltip should be anchored to.
- New settings added to `Exporting` for formatting durations: `durationFields`, `durationFormat`, and `durationUnit`.
- `showTooltipOn` (default: `"hover"`) added to `Sprite`. Another option is `"always"` which will show tooltip without any interaction with the element. [More info](https://www.amcharts.com/docs/v5/concepts/common-elements/tooltips/#Sticky_tooltips).

### Changed
- `Hierarchy` will now set bottom-level node's `cursorOverStyle: "default"` via theme, instead of hardocding it, so that it can be overridden via node templates.

### Fixed
- Stacked `XYSeries` with missing values and `valueYShow` set to `"valueYTotalPercent"` or `valueXShow` set to `"valueXTotalPercent"` were not showing columns/lines.
- Changing legend position from bottom/top to right/left was not arranging elements properly.
- If `Slice` of a `PieSeries` had a `templateField` set, legend marker was not using all the settings of that slice properly.
- Bullets were causing performance issues when data grouping was turned on.
- `Label` underline was not inheriting label's color properly.
- `Label` underline was not beign placed properly on center/right-aligned labels.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by morrow95** [comment]
@martynasma  Thank you - works fine again with the new version!
